### DISCUSSION TITLE
What were the previous 1.5 settings?

### DISCUSSION THREAD

**Discussion by lgooders** [TheLastBen/fast-stable-diffusion#1444] [Q&A]
I ask this question because I had trained a model on 1.5, 56 images (not a ton of variation) for 4500 steps, and it came out almost perfect.  I can't get anywhere close to that now and have tried multiple ways, 1.5, V2 512, V2 718.  That's even me trying to recreate with the same data set.

So my question is, what are the unet settings and the text encoder settings that can replicate the previous 1.5 version?

**Comment by TheLastBen** [comment]
unet and text encoder learning rate 2e-6, that's the only setting that changed, as for the step count, you do the same as you did before

**Reply by lgooders** [reply]
Thanks but what about the step count for text encoder?  I've found it can drastically change the results.  What is the relation to previous version?

**Reply by TheLastBen** [reply]
it's the same
